### PR TITLE
Add Prefect ETL example

### LINKED ISSUE #72: Add Prefect example?
@cicdw any interest in adding an example notebook building a Prefect workflow to examples.dask.org ?  https://github.com/dask/dask-examples#contributing

This site gets decent traffic, and we provide testing, HTML static output generation, and serving as a binder.

### PR DESCRIPTION
**Pull request by cicdw**
Closes #72 

Let me know if I need to add any more background / explanation / etc!

### CODE REVIEW

**Comment by mrocklin** [comment]
https://mybinder.org/v2/gh/cicdw/dask-examples/prefect?urlpath=lab

**Comment by cicdw** [comment]
Awesome! Just ran through it, everything runs as expected.

File: applications/prefect-etl.ipynb [hunk 1]
```diff
@@ -325,7 +325,7 @@
    "source": [
     "from IPython.display import HTML\n",
     "\n",
-    "HTML('<img src=\"./clip.gif\">')"
+    "HTML('<img src=\"./clip.gif\" alt=\"Rick Daskley\">')"
```
Comment (mrocklin) [inline applications/prefect-etl.ipynb]: Oof!

Comment (cicdw) [reply applications/prefect-etl.ipynb]: 😂

**Comment by mrocklin** [comment]
In general I think that this is great.  If I could ask for one thing, it would be to provide a few sentences at the very top describing prefect, and how it builds on Dask, maybe something like the following (though you would be better at this than I)

> Prefect is a data engineering platform that builds on top of Dask to provide features like X, Y, and Z, which are commonly needed in ETL or ABC situations.  In this example, we see ...

**Comment by cicdw** [comment]
@mrocklin updated - let me know if my intro paragraph is appropriate / sufficient.  Happy to update it further if I'm missing anything.

**Comment by cicdw** [comment]
@mrocklin just wanted to circle back on this -> anything I need to do?  No rush on my end, just wanted to make sure.

**Comment by mrocklin** [comment]
Whoops!  Yes.  Merging!

**Pull request merged**